modules/publicize/publicize-jetpack.php

```diff
@@ -464,7 +464,7 @@ function save_publicized( $post_ID, $post, $update ) {
 				 *
 				 * @param int The post ID
 				 */
-				do_action( 'jetpack_publicize_post', $post->ID );
+				do_action_deprecated( 'jetpack_publicize_post', $post->ID, '4.8.0', 'jetpack_published_post_flags' );
```

Comment:
Should we add the hook to our own list of deprecated hooks as well?
https://github.com/Automattic/jetpack/blob/master/class.jetpack.php#L5821